dnn/utils/dataLoader.py

```diff
@@ -0,0 +1,60 @@
+import tensorflow as tf
+from absl.flags import FLAGS
+from dnn.data.augmentation.detection import DetectionAugmentor
+from dnn.data.augmentation.image import ImageAugmentor
+
+
+FEATURES = {
+    'image': tf.io.FixedLenFeature([], tf.string),
+    'class_n': tf.io.FixedLenFeature([], tf.string),
+    'bbox_yxyx_n4': tf.io.FixedLenFeature([], tf.string),
+}
+
+
+class dataLoader(tf.Module):
```

Comment:
I don't think it's necessary to make it a `tf.Module`. But it does not hurt in any way.